pkgs/development/python-modules/meraki/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, python3
+}:
+
+with python3.pkgs;
+
+buildPythonPackage rec {
+  pname = "meraki";
+  version = "1.30.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-s26xGwWSWB+qpOTUe8IYo53ywYOaaUWjDznFqpmRlak=";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+    requests
+    setuptools
+  ] ++ typer.optional-dependencies.all;
+
+  pythonImportsCheck = [
+    "meraki"
+  ];
+
+  meta = with lib; {
+    description = "Provides all current Meraki dashboard API calls to interface with the Cisco Meraki cloud-managed platform.";
```

```suggestion
    description = "Provides all current Meraki dashboard API calls to interface with the Cisco Meraki cloud-managed platform";
```

No period at the end of descriptions. See https://nixos.org/manual/nixpkgs/unstable/#var-meta-description